### ISSUE TITLE
Add accessibility labels for interactive SVG thumbnails

### ISSUE THREAD

**Issue by Facelessism** [Facelessism/cradle#727]
## Description

Interactive project thumbnails should expose appropriate roles and accessible names. Add validation and fix thumbnails that cannot currently be understood or operated by assistive technologies.

**Comment by MILAN-123865** [comment]
hey @Facelessism can you please assign me?
